servicepulse/intro-archived-messages.md

```diff
@@ -11,6 +11,8 @@ reviewed: 2016-12-15
 
 In Versions 1.5.0 and above a screen to manage archived messages has been introduced in response to changes in ServiceControl relating to Archived Message retention and disposal.
 
+Archiving can be useful for failed messages that no longer have business value. Once a message is archived it can be automatically cleaned up according to the configuration of the retention policy.
```

Comment:
@sergioc - can you add a link to the document that shows the user how to configure the retention policy?